build_all.sh

```diff
@@ -8,6 +8,10 @@
 
 set -eux
 
+target=hera
```

Comment:
@HenryJuang-NOAA You don't need to set 'target' or 'compiler' here.  It will do that automatically below.

Reply:
Right. I remove it.